__docs__/wonder-blocks-link/link.stories.tsx

```diff
@@ -434,7 +502,7 @@ Inline.play = async ({canvasElement}) => {
 export const InlineLight: StoryComponentType = () => (
     <Body style={{color: Color.white}}>
         This is an inline{" "}
-        <Link href="#" inline={true} light={true}>
+        <Link href="#" inline={true} light={true} testId="link-inline-light">
```

Comment:
**nit:** I think this is no longer needed as `getByRole` is now being used (don't need to changed if you feel so).

Reply:
Ah, good catch! Thanks